Ang-Li/Weekly-Meetings.md

```diff
@@ -1,5 +1,6 @@
 # Weekly Meetings
 
+* [13th Aug 2019](#date-19th-June-2019)
```

Comment:
The links need to all be in lower case - sorry I missed this before.

Also the first one (and a couple of the others) are point to the wrong heading 😬 
 
```suggestion
* [13th Aug 2019](#date-13th-aug-2019)
```